applications/ShallowWaterApplication/custom_strategies/flux_corrected_shallow_water_scheme.h

```diff
@@ -20,7 +20,6 @@
 // Project includes
 #include "shallow_water_residual_based_bdf_scheme.h"
 #include "processes/find_global_nodal_neighbours_process.h"
-#include "utilities/parallel_utilities.h"
```

Comment:
actually you are using this, why do you remove it?

Reply:
it's included in the base class. Better to add again?

Reply:
Ah ok didn’t know that
Guess it is fine then